src/averageAges.js

```diff
@@ -15,11 +15,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(({ sex, died }) => (
+    century
+      ? sex === 'm' && century === Math.ceil(died / 100)
+      : sex === 'm'
+  ));
```

Comment:
fix in all cases pls

```suggestion
  const men = people.filter(({ sex, died }) => (century
    ? sex === 'm' && century === Math.ceil(died / 100)
    : sex === 'm'
  ));
```

Reply:
Fixed